radixdlt/src/main/java/com/radixdlt/consensus/VoteData.java

```diff
@@ -0,0 +1,78 @@
+package com.radixdlt.consensus;
```

Comment:
Missing header comment.  *Please* configure your IDE to put these in automatically for new files.  @flotothemoon or @Sajjon can help you with IntelliJ config if you need.

Reply:
yup, sorry thought i had it set up but apparently not